package.json

```
@@ -4,19 +4,23 @@
   "license": "MIT",
   "scripts": {
     "angular:build-generator": "cd node_modules/devextreme-angular-generator && npm i --no-audit --no-fund && npm run build:local",
-    "angular:discover-metadata": "npm run tools:discover-declarations && npm run tools:generate-imd -- --output-path ./packages/devextreme-angular/metadata/integration-data.json",
-    "angular:generate": "npm run angular:discover-metadata && npm run angular:build-generator && cd packages/devextreme-angular && npm run generate",
+    "angular:copy-metadata": "node ./build/copy-metadata.js NGMetaData.json ./packages/devextreme-angular/metadata/",
```

Let's use https://www.npmjs.com/package/shx instead of own implementation (shx is a CLI wrapper for shelljs which we already use)

```suggestion
    "angular:copy-metadata": shx cp NGMetaData.json ./packages/devextreme-angular/metadata/",
```